alerts/resource_alerts.libsonnet

```diff
@@ -223,6 +225,22 @@ local utils = import '../lib/utils.libsonnet';
               summary: 'Processes experience elevated CPU throttling.',
             },
           },
+          {
+            alert: 'KubeEvictionRateHigh',
+            expr: |||
+              sum(rate(kubelet_evictions{%(kubeletSelector)s}[15m])) by (%(clusterLabel)s) > %(highEvictionRateThreshold)s
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            'for': '1m',
```

Comment:
Just a thought around the alert trigger, currently:

- per-second rate over last 15 minutes
- `for` 1 minute

Means that this alert will always fire for a minimum of 15 minutes before becoming resolved, even if just one instance of evictions occurs. Is that the desired behaviour?

There might be more control/tuning possible for the sensitivity, if the per-second rate over the last 1 minute is used, in conjunction with a `for` value of whatever length of time is appropriate (`1m` for immediate, or larger to detect a sustained eviction rate).

Not saying it's wrong as it is, but wanted to clarify that the behaviour is intentional.